server/lib/payments.js

```diff
@@ -49,43 +49,44 @@ export const createPayment = (payload) => {
     return Promise.reject(new Error('Payment amount must be at least $0.50'));
   }
 
- // fetch Stripe Account and get or create Payment Method
+  // fetch Stripe Account and get or create Payment Method
   return Promise.props({
     stripeAccount: group.getStripeAccount(),
     paymentMethod: models.PaymentMethod.getOrCreate({
       token: token,
       service: 'stripe',
-      UserId: user.id })
+      UserId: user.id
     })
-  .then(results => {
-    const stripeAccount = results.stripeAccount;
-    if (!stripeAccount || !stripeAccount.accessToken) {
-      return Promise.reject(new Error(`The host for the collective slug ${group.slug} has no Stripe account set up`));
-    } else if (process.env.NODE_ENV !== 'production' && _.includes(stripeAccount.accessToken, 'live')) {
-      return Promise.reject(new Error(`You can't use a Stripe live key on ${process.env.NODE_ENV}`));
-    } else {
-      paymentMethod = results.paymentMethod;
-      return Promise.resolve();
-    }
   })
+    .then(results => {
+      const stripeAccount = results.stripeAccount;
+      if (!stripeAccount || !stripeAccount.accessToken) {
+        return Promise.reject(new Error(`The host for the collective slug ${group.slug} has no Stripe account set up`));
+      } else if (process.env.NODE_ENV !== 'production' && _.includes(stripeAccount.accessToken, 'live')) {
+        return Promise.reject(new Error(`You can't use a Stripe live key on ${process.env.NODE_ENV}`));
+      } else {
+        paymentMethod = results.paymentMethod;
+        return Promise.resolve();
+      }
+    })
 
-  // create a new subscription
-  // (this needs to happen first, because of hook on Donation model)
-  .then(() => {
-    if (isSubscription) {
-      title = description || capitalize(`${interval}ly donation to ${group.name}`);
-      return models.Subscription.create({
-        amount,
-        currency,
-        interval
-      })
-    } else {
-      title = description || `Donation to ${group.name}`
-      return Promise.resolve();
-    }
-  })
-  // create a new donation
-  .then(subscription => models.Donation.create({
+    // create a new subscription
+    // (this needs to happen first, because of hook on Donation model)
+    .then(() => {
```

Comment:
in many places, your indentation is wrong @asood123. You should check your editor.
it makes it difficult to see in the diff what really changed.